src/client.rs

```diff
@@ -142,8 +142,56 @@ pub struct CachedFeature {
     // on submission will be the next logical progression.
     enabled: AtomicU64,
     disabled: AtomicU64,
+    disabled_variant_count: AtomicU64,
     // Variants for use with get_variant
-    variants: Vec<api::Variant>,
+    variants: Vec<CachedVariant>,
+}
+
+impl CachedFeature {
+    #[allow(dead_code)]
```

Comment:
What causes this to be reported as dead code?

Reply:
It's not currently being used outside of the tests, because there is no implementation of serializing the data yet. I'll remove this when I add the variants to the metrics payload in the next PR.